### ISSUE TITLE
Can't set value

### ISSUE THREAD

**Issue by guischafer** [iulian-radu-at/react-select-material-ui#12]
```json
{
"dependencies": {
    "@material-ui/core": "^3.9.2",
    "react-select-material-ui": "^4.1.0",
}
}
```

I'm not being able to use this component in a controlled state.

```javascript
const [form, setForm] = useState({ state: ' ' });

const selectChange = state => {
    setForm({ ...form, state });
  };

<ReactSelectMaterialUi
  value={form.state}
  classes={{ root: styles.select }}
  options={selectOptions}
  placeholder='State'
  onChange={selectChange}
/>
```
Since I have to change it programmatically, I'm using it in an uncontrolled state and using a key to force a re-render, like that:

```javascript
<ReactSelectMaterialUi
  defaultValue={form.state}
  key={form.state}
  classes={{ root: styles.select }}
  options={selectOptions}
  placeholder='State'
  onChange={selectChange}
/>
```

**Comment by philstar4** [comment]
I seem to be having the same problem.  I pass in the value on initial load (tried both string and number representation), but this.state.selectedOption is never defined.

**Comment by iulian-radu-at** [comment]
Hi,
I will try to reproduce you problem and provide a fix as soon as possible.
Thanks for reporting it :)
Iulian

**Comment by pedroslopez** [comment]
I am also unable to set the initial value for multiselect. Thanks for your work.

**Comment by iulian-radu-at** [comment]
Hi Pedro, Hi all,
I fixed the problem in v4.1.1. I also added a new demo in storybook for this case (see 'Advanced tests' section).
Please check if it works for you and in case there are still problems, please let me know.
I wish you all a great day,
Iulian

**Comment by pedroslopez** [comment]
All good now, thanks!
